also reflect changed log defaults in config files and docs
https://bugzilla.sudo.ws/show_bug.cgi?id=1023

--- sudo-1.9.9/docs/sudo.conf.man.in
+++ sudo-1.9.9/docs/sudo.conf.man.in
@@ -588,7 +588,7 @@ Examples:
 .nf
 .sp
 .RS 6n
-Debug sudo /var/log/sudo_debug all@warn,plugin@info
+Debug sudo /var/share/sudo/sudo_debug all@warn,plugin@info
 .RE
 .fi
 .PP
@@ -600,7 +600,7 @@ level for the plugin subsystem.
 .nf
 .sp
 .RS 6n
-Debug sudo_intercept.so /var/log/intercept_debug all@debug
+Debug sudo_intercept.so /var/share/sudo/intercept_debug all@debug
 .RE
 .fi
 .PP
@@ -869,8 +869,8 @@ # Subsystems vary based on the program;
 # Priority may be crit, err, warn, notice, diag, info, trace, or debug.
 # Multiple subsystem@priority may be specified, separated by a comma.
 #
-#Debug sudo /var/log/sudo_debug all@debug
-#Debug sudoers.so /var/log/sudoers_debug all@debug
+#Debug sudo /var/share/sudo/sudo_debug all@debug
+#Debug sudoers.so /var/share/sudo/sudoers_debug all@debug
 .RE
 .fi
 .SH "SEE ALSO"
--- sudo-1.9.9/docs/sudo.conf.mdoc.in
+++ sudo-1.9.9/docs/sudo.conf.mdoc.in
@@ -536,7 +536,7 @@ .Pq Ql \&, .
 .Pp
 Examples:
 .Bd -literal -offset indent
-Debug sudo /var/log/sudo_debug all@warn,plugin@info
+Debug sudo /var/share/sudo/sudo_debug all@warn,plugin@info
 .Ed
 .Pp
 would log all debugging statements at the
@@ -545,7 +545,7 @@ level and higher in addition to those at
 .Em info
 level for the plugin subsystem.
 .Bd -literal -offset indent
-Debug sudo_intercept.so /var/log/intercept_debug all@debug
+Debug sudo_intercept.so /var/share/sudo/intercept_debug all@debug
 .Ed
 .Pp
 would log all debugging statements, regardless of level, for the
@@ -798,8 +798,8 @@ # Subsystems vary based on the program;
 # Priority may be crit, err, warn, notice, diag, info, trace, or debug.
 # Multiple subsystem@priority may be specified, separated by a comma.
 #
-#Debug sudo /var/log/sudo_debug all@debug
-#Debug sudoers.so /var/log/sudoers_debug all@debug
+#Debug sudo /var/share/sudo/sudo_debug all@debug
+#Debug sudoers.so /var/share/sudo/sudoers_debug all@debug
 .Ed
 .Sh SEE ALSO
 .Xr sudo_plugin @mansectform@ ,
--- sudo-1.9.9/docs/sudo_logsrvd.conf.man.in
+++ sudo-1.9.9/docs/sudo_logsrvd.conf.man.in
@@ -855,8 +855,8 @@ #connect_timeout = 30
 
 # The directory to store messages in before they are sent to the relay.
 # Messages are stored in wire format.
-# The default value is /var/log/sudo_logsrvd.
-#relay_dir = /var/log/sudo_logsrvd
+# The default value is /var/share/sudo/sudo_logsrvd.
+#relay_dir = /var/share/sudo/sudo_logsrvd
 
 # The number of seconds to wait after a connection error before
 # making a new attempt to forward a message to a relay host.
@@ -914,7 +914,7 @@ #tls_dhparams = /etc/ssl/sudo/logsrvd_dh
 [iolog]
 # The top-level directory to use when constructing the path name for the
 # I/O log directory.  The session sequence number, if any, is stored here.
-#iolog_dir = /var/log/sudo-io
+#iolog_dir = /var/share/sudo/sudo-io
 
 # The path name, relative to iolog_dir, in which to store I/O logs.
 # Note that iolog_file may contain directory components.
@@ -1003,7 +1003,7 @@ #server_facility = daemon
 [logfile]
 # The path to the file-based event log.
 # This path must be fully-qualified and start with a '/' character.
-#path = /var/log/sudo
+#path = /var/share/sudo/sudo.log
 
 # The format string used when formatting the date and time for
 # file-based event logs.  Formatting is performed via strftime(3) so
--- sudo-1.9.9/docs/sudo_logsrvd.conf.mdoc.in
+++ sudo-1.9.9/docs/sudo_logsrvd.conf.mdoc.in
@@ -783,8 +783,8 @@ #connect_timeout = 30
 
 # The directory to store messages in before they are sent to the relay.
 # Messages are stored in wire format.
-# The default value is /var/log/sudo_logsrvd.
-#relay_dir = /var/log/sudo_logsrvd
+# The default value is /var/share/sudo/sudo_logsrvd.
+#relay_dir = /var/share/sudo/sudo_logsrvd
 
 # The number of seconds to wait after a connection error before
 # making a new attempt to forward a message to a relay host.
@@ -842,7 +842,7 @@ #tls_dhparams = /etc/ssl/sudo/logsrvd_dh
 [iolog]
 # The top-level directory to use when constructing the path name for the
 # I/O log directory.  The session sequence number, if any, is stored here.
-#iolog_dir = /var/log/sudo-io
+#iolog_dir = /var/share/sudo/sudo-io
 
 # The path name, relative to iolog_dir, in which to store I/O logs.
 # Note that iolog_file may contain directory components.
@@ -931,7 +931,7 @@ #server_facility = daemon
 [logfile]
 # The path to the file-based event log.
 # This path must be fully-qualified and start with a '/' character.
-#path = /var/log/sudo
+#path = /var/share/sudo/sudo.log
 
 # The format string used when formatting the date and time for
 # file-based event logs.  Formatting is performed via strftime(3) so
--- sudo-1.9.9/docs/sudo_plugin_python.man.in
+++ sudo-1.9.9/docs/sudo_plugin_python.man.in
@@ -1613,12 +1613,12 @@ sudo.conf(@mansectform@)
 with the program set to
 \fIpython_plugin.so\fR.
 For example, to store debug output in
-\fI/var/log/sudo_python_debug\fR,
+\fI/var/share/sudo/sudo_python_debug\fR,
 use a line like the following:
 .nf
 .sp
 .RS 6n
-Debug python_plugin.so /var/log/sudo_python_debug \e
+Debug python_plugin.so /var/share/sudo/sudo_python_debug \e
     plugin@trace,c_calls@trace
 .RE
 .fi
@@ -1633,7 +1633,7 @@ calls, use:
 .nf
 .sp
 .RS 6n
-Debug python_plugin.so /var/log/sudo_python_debug plugin@trace
+Debug python_plugin.so /var/share/sudo/sudo_python_debug plugin@trace
 .RE
 .fi
 .PP
@@ -1741,7 +1741,7 @@ ModulePath=@prefix@/share/doc/sudo/examp
     ClassName=DebugDemoPlugin
 
 Debug python_plugin.so \e
-    /var/log/sudo_python_debug plugin@trace,c_calls@trace
+    /var/share/sudo/sudo_python_debug plugin@trace,c_calls@trace
 .RE
 .fi
 .SS "Option conversion API"
--- sudo-1.9.9/docs/sudo_plugin_python.mdoc.in
+++ sudo-1.9.9/docs/sudo_plugin_python.mdoc.in
@@ -1300,10 +1300,10 @@ .Xr sudo.conf @mansectform@
 with the program set to
 .Pa python_plugin.so .
 For example, to store debug output in
-.Pa /var/log/sudo_python_debug ,
+.Pa /var/share/sudo/sudo_python_debug ,
 use a line like the following:
 .Bd -literal -offset indent
-Debug python_plugin.so /var/log/sudo_python_debug \e
+Debug python_plugin.so /var/share/sudo/sudo_python_debug \e
     plugin@trace,c_calls@trace
 .Ed
 .Pp
@@ -1315,7 +1315,7 @@ For example to just see the debug output
 .Fn sudo.debug
 calls, use:
 .Bd -literal -offset indent
-Debug python_plugin.so /var/log/sudo_python_debug plugin@trace
+Debug python_plugin.so /var/share/sudo/sudo_python_debug plugin@trace
 .Ed
 .Pp
 See
@@ -1402,7 +1402,7 @@ ModulePath=@prefix@/share/doc/sudo/examp
     ClassName=DebugDemoPlugin
 
 Debug python_plugin.so \e
-    /var/log/sudo_python_debug plugin@trace,c_calls@trace
+    /var/share/sudo/sudo_python_debug plugin@trace,c_calls@trace
 .Ed
 .Ss Option conversion API
 The Python plugin API includes two convenience functions to
--- sudo-1.9.9/docs/sudoers.man.in
+++ sudo-1.9.9/docs/sudoers.man.in
@@ -5644,7 +5644,7 @@ \fIlogfile\fR
 option is set,
 \fBsudoers\fR
 will log to a local file, such as
-\fI/var/log/sudo\fR.
+\fI/var/share/sudo/sudo.log\fR.
 When logging to a file,
 \fBsudoers\fR
 uses a format similar to
@@ -6042,7 +6042,7 @@ Defaults		syslog=auth,runcwd=~
 Defaults>root		!set_logname
 Defaults:FULLTIMERS	!lecture,runchroot=*
 Defaults:millert	!authenticate
-Defaults@SERVERS	log_year, logfile=/var/log/sudo.log
+Defaults@SERVERS	log_year, logfile=/var/share/sudo/sudo.log
 Defaults!PAGERS		noexec
 .RE
 .fi
@@ -6937,7 +6937,7 @@ For example:
 .nf
 .sp
 .RS 0n
-Debug sudoers.so /var/log/sudoers_debug match@info,nss@info
+Debug sudoers.so /var/share/sudo/sudoers_debug match@info,nss@info
 .RE
 .fi
 .PP
--- sudo-1.9.9/docs/sudoers.mdoc.in
+++ sudo-1.9.9/docs/sudoers.mdoc.in
@@ -5259,7 +5259,7 @@ .Em logfile
 option is set,
 .Nm
 will log to a local file, such as
-.Pa /var/log/sudo .
+.Pa /var/share/sudo/sudo.log .
 When logging to a file,
 .Nm
 uses a format similar to
@@ -5609,7 +5609,7 @@ Defaults		syslog=auth,runcwd=~
 Defaults>root		!set_logname
 Defaults:FULLTIMERS	!lecture,runchroot=*
 Defaults:millert	!authenticate
-Defaults@SERVERS	log_year, logfile=/var/log/sudo.log
+Defaults@SERVERS	log_year, logfile=/var/share/sudo/sudo.log
 Defaults!PAGERS		noexec
 .Ed
 .Pp
@@ -6408,7 +6408,7 @@ .El
 .Pp
 For example:
 .Bd -literal
-Debug sudoers.so /var/log/sudoers_debug match@info,nss@info
+Debug sudoers.so /var/share/sudo/sudoers_debug match@info,nss@info
 .Ed
 .Pp
 For more information, see the
--- sudo-1.9.9/examples/sudo.conf.in
+++ sudo-1.9.9/examples/sudo.conf.in
@@ -135,5 +135,5 @@ # Subsystems vary based on the program;
 # Priority may be crit, err, warn, notice, diag, info, trace, or debug.
 # Multiple subsystem@priority may be specified, separated by a comma.
 #
-#Debug sudo /var/log/sudo_debug all@debug
-#Debug sudoers.so /var/log/sudoers_debug all@debug
+#Debug sudo /var/share/sudo/sudo_debug all@debug
+#Debug sudoers.so /var/share/sudo/sudoers_debug all@debug
--- sudo-1.9.9/examples/sudo_logsrvd.conf
+++ sudo-1.9.9/examples/sudo_logsrvd.conf
@@ -86,8 +86,8 @@ #connect_timeout = 30
 
 # The directory to store messages in before they are sent to the relay.
 # Messages are stored in wire format.
-# The default value is /var/log/sudo_logsrvd.
-#relay_dir = /var/log/sudo_logsrvd
+# The default value is /var/share/sudo/sudo_logsrvd.
+#relay_dir = /var/share/sudo/sudo_logsrvd
 
 # The number of seconds to wait after a connection error before
 # making a new attempt to forward a message to a relay host.
@@ -145,7 +145,7 @@ #tls_dhparams = /etc/ssl/sudo/logsrvd_dh
 [iolog]
 # The top-level directory to use when constructing the path name for the
 # I/O log directory.  The session sequence number, if any, is stored here.
-#iolog_dir = /var/log/sudo-io
+#iolog_dir = /var/share/sudo/sudo-io
 
 # The path name, relative to iolog_dir, in which to store I/O logs.
 # Note that iolog_file may contain directory components.
@@ -235,7 +235,7 @@ #server_facility = daemon
 [logfile]
 # The path to the file-based event log.
 # This path must be fully-qualified and start with a '/' character.
-#path = /var/log/sudo
+#path = /var/share/sudo/sudo.log
 
 # The format string used when formatting the date and time for
 # file-based event logs.  Formatting is performed via strftime(3) so
diff -r 9d2599418b01 examples/sudoers
--- a/examples/sudoers	Fri Feb 11 17:04:08 2022 +0100
+++ b/examples/sudoers	Fri Feb 11 17:04:52 2022 +0100
@@ -12,7 +12,7 @@
 Defaults>root		!set_logname
 Defaults:FULLTIMERS	!lecture,runchroot=*
 Defaults:millert	!authenticate
-Defaults@SERVERS	log_year, logfile=/var/log/sudo.log
+Defaults@SERVERS	log_year, logfile=/var/share/sudo/sudo.log
 Defaults!PAGERS		noexec
 
 ##
